modules/core/src/main/java/org/apache/ignite/spi/encryption/keystore/KeystoreEncryptionSpi.java

```diff
@@ -332,6 +302,22 @@ private void doEncryption(ByteBuffer data, Cipher cipher, Serializable key, Byte
         return BLOCK_SZ;
     }
 
+    /** {@inheritDoc} */
+    @Override public String getMasterKeyId() {
```

Comment:
Why do we need new method `getMasterKeyId` instead of `getMasterKeyName`?
Let's keep existing method and not deprecate it.

Reply:
Done.